tests/v2/integration/setup/main.go

```diff
@@ -31,6 +31,7 @@ func main() {
 	rancherConfig.AdminToken = token.Token
 	rancherConfig.Host = hostURL
 	rancherConfig.Cleanup = &cleanup
+	rancherConfig.ClusterName = "local"
```

Comment:
Is this necessary? I don't want to limit our integration setup to just local, as we have plans of having an imported k3d cluster work in our integration test. I don't think this is necessary.